cotk/metric/metric.py

```diff
@@ -63,9 +63,11 @@ def forward(self, data):
 			data (dict): A dict at least contains the following keys.
 			data[reference_allvocabs_key] (list of list of list): Reference sentences.
 				Does not contain start token (eg: ``<go>``) and end token (eg: ``<eos>``).
-				Outermost list: batch_size
-				Innermost list: number of words, allow different sizes
-				Second innermost list: number of sentences, allow different sizes
+				First dimension: batch_size
+				Second dimmension: number of sentences, allow different sizes for different batches
+				Third dimmension: number of words, allow different sizes for different sentences
```

Comment:
Maybe you can use the expression like [batch_size, ~sent_num, ~word_num] and say '~' means different size